pkg/chains/formats/slsa/v2alpha2/internal/resolved_dependencies/resolved_dependencies.go

```diff
@@ -116,6 +125,73 @@ func PipelineRun(ctx context.Context, pro *objects.PipelineRunObject) ([]v1.Reso
 	if err != nil {
 		return nil, err
 	}
+
+	// add on-cluster pipeline and tasks
+	rd, err := addLocalPipelineRunRefs(pro)
+	if err != nil {
+		return nil, err
+	}
+	resolvedDependencies = append(resolvedDependencies, rd...)
+	return resolvedDependencies, nil
+}
+
+// addLocalTaskRunRef adds the spec of on-cluster referenced task used by the taskrun
+func addLocalTaskRunRef(tro *objects.TaskRunObject) ([]v1.ResourceDescriptor, error) {
```

Comment:
This function is only adding one entry to the resolvedDeps list. Why do we want to return a slice?

Reply:
fixed. An effect of copy and pasting.